src/content/en/2020/seo.md

```diff
@@ -83,183 +84,227 @@ Response Code | All Devices | Mobile | Desktop
 
-When analyzing the usage of the disallow statement in robots.txt by using Lighthouse-powered data of over 6 million sites, it was found that 97.84% of them were completely crawlable, with only 1.05% using a disallow statement.
+This is notable as [Google documentation](https://developers.google.com/search/docs/advanced/robots/intro) states that site owners should not use `robots.txt` as a means to hide web pages from Google Search, as internal linking with descriptive text could result in the page being indexed without a crawler visiting the page. Instead, site owners should use other methods, like a `noindex` directive via meta robots.
+{# TODO(authors): Tie this notable fact back to the data: is it notable because the disallow numbers are so low? What does that say about site owners following Google's guidance? #}
 
-An analysis of the robots.txt disallow statement usage along the meta robots indexability directives was also done, finding 1.02% of the sites including a disallow statement along indexable pages -featuring a meta robots index tag-, with only 0.03% of sites using the disallow statement in robots.txt along noindexed page via the meta robots noindex directive.
+#### Meta robots
 
-This is notable as [Google documentation](https://developers.google.com/search/docs/advanced/robots/intro) states sites should not use robots.txt as a means to hide web pages from Google Search as internal linking with descriptive text could result in the page being indexed without a crawler visiting the page, and use instead other methods, like a `noindex` directive via meta robots.
+The `robots` meta tag and `X-Robots-Tag` HTTP header are an extension of the proposed [Robots Exclusion Protocol](https://webmasters.googleblog.com/2019/07/rep-id.html) (REP), which allows directives to be configured at a more granular level. Directive support varies by search engine as REP is not yet an official internet standard.
 
-#### Meta Robots
-
-Robots meta tag and X-Robots-Tag in HTTP headers are an extension of the proposed [Robots Exclusion Protocol](https://webmasters.googleblog.com/2019/07/rep-id.html) (REP) which allow directives to be allowed at a more granular level. Directive support varies by search engine as REP is not yet an official internet standard.
-
-Meta tags were the dominant method of granular execution with 27.70% of desktop and 27.96% of mobile pages using the tag. X-Robots-Tag directives were found on 0.27% and 0.40% of desktop and mobile respectively.
+Meta tags were the dominant method of granular execution with 27.70% of desktop and 27.96% of mobile pages using the tag. `X-Robots-Tag` directives were found on 0.27% and 0.40% of desktop and mobile, respectively.
 
+{# TODO(analysts): Should "X-Robots" be "X-Robots-Tag" here? #}
```

Comment:
@Tiggerito

Reply:
Yes. Looks like it's already fixed.

Reply:
@rviscomi sorry, didn't check the images. @antoineeripret would this be something easy for you to fix? I don't know the process of uploading the charts.

Reply:
@Tiggerito @rviscomi : name updated in the source sheet and new PNG file generated (will commit changes once I've seen all comments).

Reply:
Thanks @antoineeripret!